src/main/java/org/enginehub/worldeditcui/fabric/ViaFabricPlusHook.java

```diff
@@ -0,0 +1,9 @@
+package org.enginehub.worldeditcui.fabric;
+
+import com.viaversion.viaversion.protocols.protocol1_13to1_12_2.Protocol1_13To1_12_2;
+
+public class ViaFabricPlusHook {
+    public static void enable() {
+        Protocol1_13To1_12_2.MAPPINGS.getChannelMappings().put("WECUI", "worldedit:cui");
```

Comment:
To fit with the rest of the code, this and the checks for the mod's existence should be in `CUINetworking#subscribeToCuiPacket`. That's where MultiConnect handling is done